proto/platform/oidc/v1/oidc.platform.proto

```diff
@@ -19,6 +19,22 @@ service SecurityTokenService {
       }
     };
   }
+
+  rpc GetAccessToken(GetAccessTokenRequest) returns (TokenPair) {
+    option (google.api.http) = {
+      post: "/sts/access_token"
+    };
+  }
+}
+
+message GetAccessTokenRequest {
```

Comment:
Could also use `Refresh` here as potentially a better name

Reply:
For consistency maybe we should use `ExchangeRefreshToken`

Reply:
Done!